portal/src/components/Materials/Material/Material.vue

```diff
@@ -80,10 +80,12 @@
             </span>
           </div>
           <div
-            v-if="material.format && material.format !== 'unknown'"
+            v-if="
+              material.technical_type && material.technical_type !== 'unknown'
```

Comment:
is het niet makkelijk om `null` te gebruiken ipv `'unknown'`? Nu zie ik overal ook een check voor `'unknown'`, maar het is natuurlijk wachten totdat iemand daar niet aan gaat denken 😄 .

Reply:
Ja, ik heb het inderdaad min of meer gedachteloos overgenomen van file_type. De gedachte is volgens mij dat iets unknown is als er wel een file is, maar een onbekend type heeft (en we dus onze mapping moeten aanvullen). Het is None/null als er helemaal geen file bestaat en we dus nooit iets zouden kunnen afleiden. Zal even met Jelmer overleggen of hij dit systeem nog wilt en als dat zo is, dan moet ik natuurlijk niet overal "unknown" teruggeven, maar None waar toepasselijk. Dus zal het nog aanpassen.